src/app/core/setup/choose-use-case/choose-use-case.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="flex-row flex-wrap gap-regular flex-grow-1-3">
+  <div class="flex-row gap-regular flex-column demo-select">
+    <h3 i18n>Select your Use Case:</h3>
+    <mat-form-field appearance="fill">
+      <mat-select
+        [(value)]="selectedUseCase"
+        (selectionChange)="onSelectionChange()"
+      >
+        @for (useCase of demoUseCases; track useCase) {
+          <mat-option
+            [value]="useCase"
+            [class.greyed-out]="useCase.locale !== locale"
+          >
+            {{ useCase.name }}
```

Comment:
seeing this now, I think we have to somehow also show the language and explain the greyed out entries ...